src/editor/opengl/OpenGL.cpp

```diff
@@ -1,16 +1,17 @@
+#include <QOpenGLWidget>
 #include "OpenGL.hpp"
 
 namespace Tungsten {
 
 namespace GL {
 
+QOpenGLFunctions_3_2_Core* glf = 0;
```

Comment:
This is very nitpicky, but can you move the * one to the right, next to glf? That's the coding style used in the rest of the project - & and * next to the variable name, not the type.